### ISSUE TITLE
Files are not deleted from S3 bucket when the object is deleted from Keystone

### ISSUE THREAD

**Issue by vonba** [keystonejs-contrib/k6-contrib#24]
When I delete an object from Keystone files are not removed. I can't see any configuration to enable this in the code, is this implemented at all?

If this isn't implemented, is there a workaround? It seems like a necessary feature to me otherwise buckets will quickly fill up with trash, will also costs money.

**Comment by gautamsi** [comment]
Not really, you can hel set this up in a PR

**Comment by gautamsi** [comment]
it is based on `aws-sdk`

**Comment by limegorilla** [comment]
This is also the case for Azure blob storage - I'm assuming it's a similar story?

**Comment by gautamsi** [comment]
right, no support for deletion. but you can always delete them by calling aws-sdk directly
